### ISSUE TITLE
Possibility inconsistent command

### ISSUE THREAD

**Issue by Pajri** [dotnet/docs#33836]
Hi, I would like to ask about this command 
`dumpheap -mt 00007faddaa50f90`

If we want to list all the String instance, why don't we use this instead?
`dumpheap -mt 00007f6c1dc00f90`

Because the **MT collumn** for Sistem.String is 00007f6c1dc00f90

```
> dumpheap -stat

Statistics:
              MT    Count    TotalSize Class Name
...
00007f6c1eeefba8      576        59904 System.Reflection.RuntimeMethodInfo
00007f6c1dc021c8     1749        95696 System.SByte[]
00000000008c9db0     3847       116080      Free
00007f6c1e784a18      175       128640 System.Char[]
00007f6c1dbf5510      217       133504 System.Object[]
00007f6c1dc014c0      467       416464 System.Byte[]
00007f6c21625038        6      4063376 testwebapi.Controllers.Customer[]
00007f6c20a67498   200000      4800000 testwebapi.Controllers.Customer
00007f6c1dc00f90   206770     19494060 System.String
Total 428516 objects
```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 2913d994-4c72-cc6e-0bd4-ba122ae21753
* Version Independent ID: 03f3761e-be31-bae9-6ac7-ad86db389212
* Content: [Debug a memory leak tutorial](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/debug-memory-leak)
* Content Source: [docs/core/diagnostics/debug-memory-leak.md](https://github.com/dotnet/docs/blob/main/docs/core/diagnostics/debug-memory-leak.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @tommcdon
* Microsoft Alias: **tommcdon**

**Comment by gewarren** [comment]
@tommcdon Can you help with this question?

**Comment by gewarren** [comment]
I agree the example should be changed. I'll fix this along with https://github.com/dotnet/docs/issues/35077 in September.
